impersonation/README.md

```
@@ -0,0 +1,92 @@
+    ```json
+    {
+      "impersonatedUserId": "<id of user to impersonate>",
+      "reason": "<reason for impersonation>"
+    }
+    ```
+
+- **Success Response of Impersonation Request:**
+
+  - **Code:** 201
+  - **Content:**
+    ```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)